eox_tenant/signals.py

```diff
@@ -176,6 +192,13 @@ def start_tenant(sender, environ, **kwargs):  # pylint: disable=unused-argument
     # Do the update
     _update_settings(domain)
 
+    # Some django apps need to be reinitialized
+    try:
+        if base_settings.EDNX_TENANT_INSTALLED_APPS:
```

Comment:
At this point we're playing with tenant settings because _update_settings was already called right?

Reply:
correct